FantasmoSDK/app/src/main/java/com/fantasmo/sdk/views/FMParkingView.kt

```diff
@@ -200,7 +206,15 @@ class FMParkingView @JvmOverloads constructor(
             statistics.visibility = View.GONE
         }
 
-        startQRScanning()
+        if (enableQRCodeScanner){
+            startQRScanning()
+        }
+        else {
+            // Set an AR anchor now
+            fmARCoreView.startAnchor()
```

Comment:
this sets the anchor to be the actual device? I think if we skip the QR code scanning, we don't want to set any anchor (we don't need one). Just making sure that's the case 😄

Reply:
When we enter a QR code manually, it sets an anchor. I followed the same principle of anchoring when we skip the QR code entirely. Don't know for sure if it's the right approach.

Reply:
There is no guarantee that the camera will be centered on or even facing the scooter if the QR code is entered manually, so I don't think that's the right approach, @lucaskuzma could you confirm?

I think if we enter the QR code manually, we just don't set an anchor? Not sure 🤔

Reply:
that or we should really prompt people and say that when the QR code is entered manually, to please point the camera at the scooter before pressing confirm ⚡

Reply:
Yeah we should just set the anchor automatically when the code is entered manually or skipped. If customers are not using the AR session during the QR code scan, this is the best we can do, and hopefully the phone has not moved too far from the QR code at this stage.

Reply:
This is a known drawback of not reading QR code in the AR session.